p11/index.html

```diff
@@ -1 +1,50 @@
-<!-- TODO -->
+<!-- TODO --> 
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>KWEB Survey Page</title>
+</head>
+<body>
+    <p>
```

Comment:
Line 10~14에는 p tag를 사용할 수 없고, div 또는 span tag로 phrasing하는 것이 적절합니다! 다음을 잠고해 주세요.
> a \<p> element cannot contain another block element, it can only contain, "phrasing content," and its end tag may be omitted if the p element is immediately followed by an address, article, aside, blockquote, div, dl, fieldset, footer, form, h1, h2, h3, h4, h5, h6, header, hgroup, hr, main, nav, ol, p, pre, section, table, or ul, element…